### PR TITLE
GUI/22/Merge GUI views and restructure frames

### PR DESCRIPTION
**Pull request by brendanehrman1**
- Remove separate profile view and reconstruct as "settings" popup
- Beautify GUI
- Add additional GUI styles

### CODE REVIEW

File: data/config.ini [hunk 1]
```diff
@@ -1,9 +1,10 @@
...
 [StudentProfile]
-name
-major
-graduation_year
-graduation_month
\ No newline at end of file
+name = Enter here.
+major = Computer Science and Engineering
+graduation_year = 2022
+graduation_month = January
```
Comment (johndpjr) [inline data/config.ini:9]: Configuration should be reset here

**Review by johndpjr — COMMENTED** [review]
Wow! This looks much, much better

One thing (other than config):
* Should we change the main color theme to maroon (hex #500000)?

**Comment by brendanehrman1** [comment]
> * Should we change the main color theme to maroon (hex #500000)?
Good point. I've centralized the color scheme within styles.py to make color changes easier now (the main color is now stored in HIGHLIGHT), and I've temporarily changed the color to "#500000". Feel free to change any of the colors again if you think it would look better!

**Review by johndpjr — APPROVED** [review]
Looks good! I'm supper glad we were able to get this in...

**Pull request merged**